infra/Pulumi.prod.yaml

```diff
@@ -0,0 +1,43 @@
+config:
+  aws-native:region: us-west-2
+  aws:region: us-west-2
+  openpipe:ANTHROPIC_API_KEY:
+    secure: AAABAG5PApLpxIrXlau1/D5VPnS7yxwv88kXog0O1T7xBxsmlcfoYWByoJ3Mw3wtM3ZXXUcrdbt/jJA5UM3Ke5Dm78lclNpoLcizDYVZ6i3+n/sRFNXXfA8musbQX6dsUAvh1Nja8mqkAp9di1IMz+eweyfHLiFPPmhrngBaD6SVc0iU9iq8dFsSPSg=
```

Comment:
Are all these keys okay to include in the repo?

Reply:
Yeah they're fully encrypted, don't see a security risk here.